File: VB/ViewModel/EFCore/InstantFeedbackMode/EditIssueInfo.vb
Imports DevExpress.Mvvm
Imports System.Collections.Generic
Imports EFCoreIssues.Issues

Public Class EditIssueInfo
    Inherits BindableBase

    Public Sub New(ByVal dbContext As IssuesContext, ByVal users As IList)
        Me.DbContext = dbContext
        Me.Users = users
    End Sub

    Public ReadOnly Property DbContext As IssuesContext
    Public ReadOnly Property Users As IList
End Class

File: CS/CodeBehind/EFCore/ServerMode/EditIssueInfo.cs
using DevExpress.Mvvm;
using System.Collections;
using EFCoreIssues.Issues;

namespace EFCoreIssues {
    public class EditIssueInfo : BindableBase {
        public EditIssueInfo(IssuesContext dbContext, IList users) {
            DbContext = dbContext;
            Users = users;
        }
        public IssuesContext DbContext { get; }
        public IList Users { get; }
    }
}